### PR TITLE
Prl 6232

### PR DESCRIPTION
**Pull request by OLT2000**
### Change description

PRL-6232 The Home

### JIRA link

[PRL-6232](https://tools.hmcts.net/jira/browse/PRL-6232)

**Before merging a pull request make sure that:**

- [x] Commits are meaningful and simple
- [x] All commits are squashed into a single commit
- [x] README and other documentation has been updated / added (if needed)

### CODE REVIEW

File: e2e/tests/manageCases/createCase/FL401/fl401TheHome.spec.ts [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    });
+  });
+
+  test(`Complete the FL401 The Home Journey with the following options:
+        Not accessibility testing, 
+        Not Error Messaging (There isn't any), 
+        Applicant doesn't have children
+        No to all booleans, 
+        'No' to has the applicant or respondent ever lived at the address, 
+        'No' do they ever intend to live at the address, @crossbrowserManagecases`, async ({
```
Comment (JakeCG) [inline e2e/tests/manageCases/createCase/FL401/fl401TheHome.spec.ts:148]: Does this need to be here

File: e2e/tests/manageCases/createCase/FL401/fl401TheHome.spec.ts [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      accessibilityTest: false,
+      applicantHasChildren: false,
+      fl401TheHomeYesNo: false,
+      fl401EverLivedAtAddress: "yesApplicant",
+      subJourney: true,
+    });
+  });
+});
+
+test(`Test the accessibility of the fl401 The Home Journey saying No to all, @accessibilityManageCases`, async ({
```
Comment (JakeCG) [inline e2e/tests/manageCases/createCase/FL401/fl401TheHome.spec.ts:199]: DELETE

**Review by JakeCG — CHANGES_REQUESTED** [review]

File: e2e/tests/manageCases/createCase/FL401/fl401TheHome.spec.ts (hunk 1 above)
Comment (OLT2000) [reply e2e/tests/manageCases/createCase/FL401/fl401TheHome.spec.ts:148]: they aren't boolean but I'll change above to 'no to all options'

**Review by Louis-Solirius — APPROVED** [review]
LGTM

**Review by JakeCG — APPROVED** [review]

**Pull request merged**